### ISSUE TITLE
Fix a11y chart tests

### ISSUE THREAD

**Issue by lukasholzer** [dynatrace-oss/barista#526]
# Bug Report

## Expected Behavior

The a11y tests should pass with our configuration.

## Current Behavior

the blacklisted examples don't pass.

```typescript
/** Blacklisted default examples */
const BLACKLIST = [
  'chart-selection-area-default-example',
  'chart-stream-example',
  'chart-loading-example',
];
```

## Steps to Reproduce

Go to the `apps/demos/src/barista-examples.a11y.ts` a11y tests file and uncomment the blacklist.
Then run `ng run demos:a11y --configuration=watch`

Use `--configurartion=ci` to execute it like on the ci

## Detailed Description

```
 ✖ chart-loading-example

   1) AssertionError: 1 violations found:
      1) ARIA attributes must conform to valid names
          "dt-loading-spinner"
      : expected false to be truthy

 ✖ chart-selection-area-default-example

   1) AssertionError: 1 violations found:
      1) ARIA attributes must conform to valid names
          "dt-chart-range"
      	"dt-chart-timestamp"
      : expected false to be truthy

 ✖ chart-stream-example

   1) AssertionError: 1 violations found:
      1) ARIA attributes must conform to valid names
          "dt-loading-spinner"
      : expected false to be truthy
```1

**Comment by tomheller** [comment]
This one depends on #524 to be merged.
